.github/workflows/main.yml

```
@@ -12,19 +12,37 @@
+      - name: Setup Chrome 🌐
+        run: |
+          chmod +x .github/workflows/setup-chrome.sh
+          .github/workflows/setup-chrome.sh
+
+      - name: Install Dependencies 📦
         run: |
           npm ci
+          npm install -g wait-on
+
+      - name: Generate Cards and Build 🔧
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:8:6: Quote this to prevent word splitting

(shellcheck)